### PR TITLE
Update it-IT.lua

### PR DESCRIPTION
**Pull request by Enryy**

### CODE REVIEW

File: languages/it-IT.lua [hunk 1]
```diff
@@ -429,814 +637,1130 @@ OP.Global.Locales.Languages["it-IT"] = {
...
-		status_reset_command_parameter_server_id = "server id",
-		status_reset_command_parameter_server_id_help = "L'ID del giocatore di cui vuoi reimpostare lo stato. Se lasciato vuoto, verrai selezionato tu automaticamente.",
+		status_reset_command_help = "Resetta lo status level.",
+		status_reset_command_parameter_server_id = "ID Server",
+		status_reset_command_parameter_server_id_help = "L' ID del giocature di cui si vuole resettare lo status server.",
 		status_reset_command_substitutes = "",
 
-		set_body_armor_command = "/set_body_armor",
-		set_body_armor_command_help = "Imposta il livello di armatura di qualcuno.",
+	    set_body_armor_command = "/set_body_armor",
```
Comment (InZidiuZ) [inline languages/it-IT.lua:1499]: Bad tabbing

File: languages/it-IT.lua [hunk 2]
```diff
@@ -1245,39 +1769,46 @@ OP.Global.Locales.Languages["it-IT"] = {
...
-		slots_already_set_to = "The server slots are already set to `${slots}`.",
-		slots_set_to = "The server slots have now been set to `${slots}`."
+		joining_the_queue = "entrando in coda...",
+		timed_out_before_joining = "Sei crashato prima di poter iniziare a entrare nel server.",
+		server_reload_while_in_loading = "Il server e' stato riavviato.",
+		server_reload_while_in_queue = "Il server e' in riavvio. Perfavore riconnetiti alla coda.",
+		took_too_long_to_connect = "Ci hai impiegato troppo a conneterti!",
+		queue_position_with_priority = "🐌 Sei ${queueEntryId}/${queueLength} nella coda con priorità ${priorityName}. 🕐${queueTime}",
+		queue_position_without_priority = "🐌 Sei ${queueEntryId}/${queueLength} in coda. 🕐${queueTime}\nStanco della coda? Sostienici per avere la priorità in coda!",
+		server_is_starting = "In attesa dell'avvio del server    ...",
```
Comment (InZidiuZ) [inline languages/it-IT.lua:1796]: Why are there a bunch of spaces here?

File: languages/it-IT.lua [hunk 3]
```diff
@@ -16,7 +16,9 @@ OP.Global.Locales.Languages["it-IT"] = {
 		invalid_input = "Input non valido.",
 		missing_input = "Missing Input.",
 		player_not_found = "Non è stato trovato nessun giocatore con questo ID `${serverId}`.",
-		something_went_wrong = "Qualcosa è andato storto. Perfarvore prova di nuovo."
+		something_went_wrong = "Qualcosa è andato storto. Per favore prova di nuovo.",
+		yes = "Yes",
+		no = "No"
```
Comment (InZidiuZ) [inline languages/it-IT.lua:21]: Were these two left out intentionally?

**Review by InZidiuZ — CHANGES_REQUESTED** [review]
Looking very good!

File: languages/it-IT.lua (hunk 1 above)
Comment (Enryy) [reply languages/it-IT.lua:1499]: Precisely which string should I adjust?

**Comment by Enryy** [comment]
I should have solved it, let me know

**Comment by Enryy** [comment]
Did you get the changes I made?

**Comment by InZidiuZ** [comment]
Not all...

File: languages/it-IT.lua [hunk 4]
```diff
@@ -3466,132 +4353,136 @@ OP.Global.Locales.Languages["it-IT"] = {
...
+		right_thigh = "Right Thigh",
+		left_thigh = "Left Thigh",
+		lower_spine = "Lower Spine",
+		center_spine = "Center Spine",
+		upper_spine = "Upper Spine",
+		root_spine = "Root Spine",
+		right_clavicle = "Right Clavicle",
+		left_clavicle = "Left Clavicle"
+	
+    }
```
Comment (InZidiuZ) [inline languages/it-IT.lua:4487]: Bad spacing

**Review by InZidiuZ — CHANGES_REQUESTED** [review]

File: languages/it-IT.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (Enryy) [reply languages/it-IT.lua]: by for bad tabbing you mean that /set_body_armor_command happens in the wrong line right?

[the remark this answers is not in the document]
Comment (InZidiuZ) [reply languages/it-IT.lua]: You use spaces instead of tabs and it's also not aligned with the other entries as it should be.

**Comment by InZidiuZ** [comment]
I ran the file through checks and found these irregularities:
```
"  " @ it-IT.lua:236
"  " @ it-IT.lua:312
"  " @ it-IT.lua:1501
"  " @ it-IT.lua:1888
"  " @ it-IT.lua:1889
"  " @ it-IT.lua:1892
"  " @ it-IT.lua:2071
"  " @ it-IT.lua:2372
"  " @ it-IT.lua:3087
"  " @ it-IT.lua:3352
"  " @ it-IT.lua:3488
"  " @ it-IT.lua:3980
"  " @ it-IT.lua:4257

Not cool occurrences found: 14
```

**Comment by Enryy** [comment]
I should have solved it now

**Comment by InZidiuZ** [comment]
You didn't even make any changes to the lines I commented on...

**Review by InZidiuZ — APPROVED** [review]
Finally!

**Pull request merged**